### DISCUSSION TITLE
Website URI for Ultimate Beneficial Owner?

### DISCUSSION THREAD

**Discussion by sorliden** [OP-TED/eForms-SDK#104] [Q&A] [answered]
Hi!

Website URI for Ultimate Beneficial Owner should be optional according to table 1.
https://docs.ted.europa.eu/eforms/1.0.0/schema/parties.html#organizationSection

However the UltimateBeneficialOwnerType doesn't contain a WebsiteURI element.
Only CompanyType, TouchPointType and PartyType contain a WebsiteURI.

What element should be used for Website URI for Ultimate Beneficial Owner?

Best regards
  Mårten Sörliden

**Comment by YvesJo** [comment, marked as answer]
Hi Mårten,
Thanks for the question. No website is expected for a UBO, and table 1 in the documentation will be updated accordingly.
KR,

**Reply by sorliden** [reply]
Thanks!
